cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraExecutor.java

```diff
@@ -184,158 +173,220 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo task) {
             final SlaveStatusDetails details = SlaveStatusDetails.newBuilder()
                 .setStatusDetailsType(SlaveStatusDetails.StatusDetailsType.ERROR_DETAILS)
                 .setSlaveErrorDetails(
-                    SlaveErrorDetails.newBuilder()
-                        .setMsg(msg)
-                        .setDetails(e.getMessage())
+                        SlaveErrorDetails.newBuilder()
+                                .setMsg(msg)
+                                .setDetails(e.getMessage() != null ? e.getMessage() : "-")
                 )
-                .build();
+                    .build();
             final TaskStatus taskStatus = taskStatus(task, TaskState.TASK_ERROR, details);
             driver.sendStatusUpdate(taskStatus);
         }
         LOGGER.debug(taskIdMarker, "< launchTask(driver : {}, task : {})", driver, protoToString(task));
     }
 
+    private void startJob(ExecutorDriver driver, TaskInfo task, TaskDetails taskDetails) {
+        if (process == null) {
+            driver.sendStatusUpdate(taskStatus(task, TaskState.TASK_ERROR, nullSlaveStatusDetails()));
```

Comment:
Added an error-code field to the protobuf message for that reason (commit 7d080c8)